engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthTest.java

```diff
@@ -32,10 +29,8 @@
  */
 public final class DisableDepthTest extends SetStateParameter {
     private static final int PARAMETER = GL_DEPTH_TEST;
```

Comment:
In line with one of your previous comment, should I delete this variable too?

(Just fyi, it's used only once in the constructor to pass the parameter to SetStateChange's constructor)